### ISSUE TITLE
ANDI and Storyline 360 modules

### ISSUE THREAD

**Issue by SLan-rgb** [SSAgov/ANDI#247]
When testing modules created in Storyline 360, it gives the alert "If <canvas> element is interactive with a mouse, it's not keyboard accessible because there is no focusable fallback content," eve though there is no canvas element. 

![Canvas flag](https://github.com/user-attachments/assets/eae6e5d8-5983-4773-b58f-198415d26ed4)

**Comment by JohnCotterSSA** [comment]
If ANDI threw that alert, then there is definitely a Canvas element in the DOM on that page.
